src/server/pkg/deploy/cmds/cmds.go

```diff
@@ -1174,18 +1179,18 @@ persistent volume was manually provisioned (i.e. if you used the
 				return err
 			}
 
-			if includingJupyterHub {
-				// remove jupyterhub
-				if err = helm.Destroy(activeContext, "jhub", namespace); err != nil {
+			if includingIDE {
+				// remove IDE
+				if err = helm.Destroy(activeContext, "pachyderm-ide", namespace); err != nil {
```

Comment:
@jdoliner because of this, `pachctl undeploy --ide` will fail if there no IDE was deployed in the first place. I'm of two minds here:
1) on the one hand, hard failing makes sense because users should be aware of something they explicitly intended not working as designed
2) on the other hand, none of the other operations in `pachctl undeploy` fail if the resources aren't actually there

I'm leaning toward changing it to (2), but wanted to get some feedback before doing so

Reply:
I think 2 is the right choice, I went through basically the same thing with `undeploy` originally, and initially went with 1, but eventually realized it was annoying, so then switched to 2.